<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="WIN-JENKINST81SCFS" name="Test.Initiate" tests="2" failures="1" timestamp="4 Sep 2017 08:34:58 GMT" time="35.640" errors="0">
  <testcase name="checkHome" time="35.603" classname="Test.Initiate">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Test.Registration.checkFields(Registration.java:59)
	at Test.Registration.executeAllTest(Registration.java:21)
	at Test.Initiate.checkRegistration(Initiate.java:44)
	at Test.Initiate.checkHome(Initiate.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- checkHome -->
  <testcase name="checkFail" time="0.037" classname="Test.Initiate"/>
</testsuite> <!-- Test.Initiate -->
